game/scripts/npc/npc_abilities_override.txt

```diff
@@ -82,7 +82,7 @@
 #base "abilities/chen_test_of_faith_teleport.txt"
 #base "abilities/clinkz_death_pact.txt"
 #base "abilities/clinkz_searing_arrows.txt"
-#base "abilities/clinkz_strafe.txt"
+#base "abilities/clinkz_strafe_oaa.txt"
```

Comment:
No reason to change this.

Reply:
Doesn't matter. IMO it shouldn't go in here since, but either way it's redundant to have it both here and in custom. You should remove one, and I would leave this one as the original ability.